lua/xcodebuild/platform/macos.lua

```diff
@@ -11,21 +11,28 @@ local M = {}
 
 ---Simply starts the application on macOS.
 ---@param appPath string
+---@param productName string
 ---@param callback function|nil
 ---@return number # job id
```

Comment:
You also have to update docs:
https://github.com/wojciech-kulik/xcodebuild.nvim/blob/main/CONTRIBUTING.md#4%EF%B8%8F%E2%83%A3-help-check

Just run: `make help-update`